packages/mui-codemod/package.json

```diff
@@ -36,7 +36,7 @@
     "jscodeshift-add-imports": "^1.0.11",
     "postcss": "^8.5.6",
     "postcss-cli": "^11.0.1",
-    "yargs": "^17.7.2"
+    "yargs": "^18.0.0"
```

Comment:
If we are fine with this slight BC, then this package `engines.node` field needs to be updated to `>=20.19`.